.github/workflows/deflake.yml

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Deflake a test'
+on:
+  workflow_dispatch:
+    inputs:
+      test-module:
+        description: 'Gradle sub-module which contains the test being de-flaked. Should be like :core'
+        required: true
+        type: string
+      test-pattern:
+        description: 'Test class to de-flake. Should be like *SomeTest*'
+        required: true
+        type: string
+      test-repeat:
+        description: 'Number of times to invoke the test'
```

Comment:
Should we remind users that it works only for `ClusterTest`/`ClusterTemplate`/`ClusterTests`?